app/views/users/show.html.erb

```diff
@@ -4,16 +4,20 @@
   </div>
 </div>
 
-<div class="row mb-2">
-  <div class="col-md-6 offset-md-3">
-    <%= render "users/profile_nav", user: @user %>
-  </div>
-</div>
+<% truevalue = current_user == @user || !@user.private? || current_user.leaders.include?(@user) %>
```

Comment:
I think this might be a type but change truevalue on line 7 to if so it can be a conditional.

Reply:
In this case, I am using truevalue as a variable for setting the boolean for the conditional statement.

Reply:
Yep understood. Thank you